docs/introduction.md

```diff
@@ -2,69 +2,146 @@
 
 ![alt](../resources/KubeVela-01.png)
 
-Welcome to KubeVela! This documentation covers why built KubeVela, what KubeVela is, and comparison of KubeVela vs. others.
-
-## Why KubeVela?
-
-The modern patterns of cloud-native application management has been merited with consistency across clouds and infrastructure by abstracting away the low level details with Kubernetes. However, this also introduced another layer of complexity: pods, ingresses, claims, service meshes, and so on. We've seen the lack of Kubernetes knowledge and its high learning curve have impacted user experience, slowed down the software shipping, led to user misconfiguration and production issues. Developers start questioning the value of cloud-native revolution, "why bother me with those YAML files?".
-
-Another challenge we've seen is in solving this problem. There are many successful platforms (including PaaS and Serverless) in the community that brought great developer experience to cloud-native world. But for platform teams in many organizations, this poses a new challenge in how to adapt or extend these platforms to meet their own needs in their own scenarios. Essentially, platforms aiming at fixing developer facing issues tend to be highly opinionated in terms of user interfaces, assumptions and implementations. This makes extending such platforms leveraging wider community (e.g. Kubernetes ecosystem) almost impossible, and many of these platforms essentially become close systems.
-
-KubeVela intends to make shipping applications more enjoyable for developers, and at the same time, provides platform engineers with full extensibility to build their own platforms leveraging the Kubernetes community.
+## Motivation
+
+The modern trends of cloud-native application management have moved towards
+pursuing a simplified developer experience across clouds and infrastructures.
+As the dominating cloud-native platform, Kubernetes, however, has brought
+significant challenges for achieving this goal. Kubernetes has a considerably
```

Comment:
Let's soften the narratives a bit, this part now reads like K8s is doing the wrong way.

Essentially k8s did very well in abstracting away the details of infra and cloud (i.e. bring consistency for application infrastructure), this makes building platforms like KubeVela super easy. But on the other hand, Kubernetes is not the end game, that's why we build KubeVela (a developers facing platform).